build.assets/Dockerfile

```diff
@@ -238,7 +238,7 @@ RUN mkdir -p helm-tarball && \
     cp helm-tarball/$(go env GOOS)-$(go env GOARCH)/helm /bin/ && \
     rm -r helm-tarball*
 # TODO(hugoShaka): remove this backward compatible hack with teleportv13 buildbox
-RUN helm plugin install https://github.com/quintush/helm-unittest && \
+RUN helm plugin install https://github.com/quintush/helm-unittest --version 0.2.11 && \
```

Comment:
Looks like the new 0.3.1 version of this plugin introduced a backwards incompatible change and removed the `-3` flag. When we bumped Go earlier today, our buildbox got rebuilt and Helm unit tests workflow started failing because of this. I've decided to pin the version on this branch, it's already pinned on v13 and master but not here for some reason.